paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt

```diff
@@ -66,17 +69,36 @@ internal class Renderer(
             }.apply { loadResources() }
           )
       } else {
-        val resourceFolderRepositories = environment.libraryResourceDirs.map {
+        val localResources = environment.localResourceDirs.map {
```

Comment:
if we're going to migrate more than just ResourceFolderRepository and AarSourceRepository, then let's revert these changes to Renderer and SessionParamsBuilder in this PR, because we'll need to then port Project/App/etc., which is fine, cuz I have it already ported!